docs/rfc/rfc-1.adoc

```diff
@@ -253,12 +253,26 @@ wallet and closes.
 
 === Wallet Lifecycle
 
-Wallets are periodically created, where the period length is a governable
-parameter. To create a new wallet, a group of 100 operators is selected from
-the pool of available operators (some operators may be selected twice if there
-are not enough) using a process called sortition. The probabiliy that a
-particular operator is chosen is based on their stake weight, which in turn is
-based on the number of `T` tokens they have invested in the staking contract.
+Governable Parameters:
+
+- `wallet_dust_threshold` the smallest amount of btc a wallet can hold before we
+  attempt to close the wallet and transfer the funds to a randomly selected
+  wallet
```

Comment:
This is only for the currently active wallet, right? If yes, worth being explicit about it.

Reply:
And if yes, why not `wallet_min_btc` to keep it aligned with `wallet_max_btc`? 🤔

Reply:
I was originally thinking that param would apply for any wallets and would be multi-purpose. For non-active wallets, if there was ever a redemption that puts us below `wallet_min_btc`, that's when we start the process to transfer the funds and close the wallet. 

Applied to the active wallet, it's what stops us from needlessly creating new wallets. I'm also okay with having two separate parameters so that we can control those two numbers separately (why not?).

Reply:
updated in f348255